docs/pages/versions/unversioned/sdk/notifications.mdx

```diff
@@ -21,6 +21,8 @@ import { PlatformTags } from '~/ui/components/Tag';
 
 > **warning** Push notifications (remote notifications) functionality provided by `expo-notifications` is unavailable in Expo Go from SDK 53. A [development build](/develop/development-builds/introduction/) is required to use push notifications. Scheduled notifications (local notifications) remain available in Expo Go.
 
+> **warning** Push notifications are not supported on Android Emulator or iOS Simulator. However, you can still use local notifications on these platforms.
```

Comment:
for me, remote notifs work in android emulator, it doesn't work in simulator only